Source/Core/DolphinQt2/Settings/GeneralPane.cpp

```diff
@@ -0,0 +1,177 @@
+#include "DolphinQt2/Settings/GeneralPane.h"
+
+#include <QCheckBox>
+#include <QComboBox>
+#include <QFormLayout>
+#include <QGroupBox>
+#include <QLabel>
+#include <QMessageBox>
+#include <QPushButton>
+#include <QSlider>
+#include <QVBoxLayout>
+#include <QWidget>
+
+#include "Core/Analytics.h"
+#include "Core/ConfigManager.h"
+
+GeneralPane::GeneralPane(QWidget* parent) : QWidget(parent)
+{
+  CreateLayout();
+  ConnectLayout();
+
+  LoadConfig();
+}
+
+void GeneralPane::CreateLayout()
+{
+  m_main_layout = new QVBoxLayout;
+  // Create layout here
+  CreateBasic();
+
+#if defined(USE_ANALYTICS) && USE_ANALYTICS
+  CreateAnalytics();
+#endif
+
+  CreateAdvanced();
+
+  m_main_layout->addStretch(1);
+  setLayout(m_main_layout);
+}
+
+void GeneralPane::ConnectLayout()
+{
+  connect(m_combobox_language, static_cast<void (QComboBox::*)(int)>(&QComboBox::activated),
+          [=](int index) { GeneralPane::OnSaveConfig(); });
```

Comment:
Capturing with = is required in Qt. Otherwise, you get compile time errors:
![image](https://cloud.githubusercontent.com/assets/866151/25784504/f819f212-3322-11e7-9bd1-d97f23f72621.png)